src/zenml/integrations/gcp/image_builders/gcp_image_builder.py

```diff
@@ -130,6 +130,9 @@ def build(
             build_context=build_context,
             parent_path_directory_name="cloud-build-contexts",
         )
+        docker_build_options = {
+            "--" + key: value for key, value in docker_build_options.items()
```

Comment:
Does this actually work? When I try running a local `docker build --rm True ...` this fails for me 🤔

While checking the code, I also saw that in `_configure_cloud_build` we create a list `docker_build_args` which converts the build options dict into a list, but that seems to be completely unused?

Reply:
I don't know why but yes https://console.cloud.google.com/cloud-build/builds;region=global/7f5ab3a5-2dd7-467e-aba4-200070db7639?project=zenml-core 😅

Reply:
> While checking the code, I also saw that in _configure_cloud_build we create a list docker_build_args which converts the build options dict into a list, but that seems to be completely unused?

Hahaha yes it is never used, instead it uses `*build_options`, which only passes in the keys, that's why my "fix" above made it just receive `["--pull", "--rm"]` without the `True` values 😄